dashboard/test/models/experiment_test.rb

```diff
@@ -1,22 +1,25 @@
 require 'test_helper'
 
 class ExperimentTest < ActiveSupport::TestCase
-  setup do
-    Experiment.stubs(:should_cache?).returns false
+  setup_all do
```

Comment:
Not sure if you need `self.use_transactional_test_case = true` with `setup_all`, seems to be the paradigm.